### PR TITLE
Example for microsoft azure attestation

### PR DESCRIPTION
**Pull request by yangfh2004**
Include an example for  using Teaclave Rust SGX SDK with Microsoft Azure Attestation

### CODE REVIEW

File: samplecode/microsoft_azure_attestation/app/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+    if qe3_ret != sgx_quote3_error_t::SGX_QL_SUCCESS {
+        println!("Error in sgx_qe_get_target_info. {:?}\n", qe3_ret);
+        return None;
+    }
+
+    //println!("target_info.mr_enclave = {:?}", ti.mr_enclave.m);
+    //println!("target_info.config_id = {:02x}", ti.config_id.iter().format(" "));
+
+    let quote_size = std::mem::size_of::<sgx_target_info_t>();
+    let mut v: Vec<u8> = vec![0; quote_size];
```
Comment (volcano0dr) [inline samplecode/microsoft_azure_attestation/app/src/main.rs]: I think  the `v` is unused.

File: samplecode/microsoft_azure_attestation/bin/maa
Comment (volcano0dr) [inline samplecode/microsoft_azure_attestation/bin/maa:1]: please delete the `maa` binary.

File: samplecode/microsoft_azure_attestation/app/src/main.rs (hunk 1 above)
Comment (yangfh2004) [reply samplecode/microsoft_azure_attestation/app/src/main.rs]: @volcano0dr removed

File: samplecode/microsoft_azure_attestation/bin/maa
Comment (yangfh2004) [reply samplecode/microsoft_azure_attestation/bin/maa:1]: @volcano0dr deleted

**Review by volcano0dr — APPROVED** [review]
LGTM

**Pull request merged**